lib/nudge_web/router.ex

```diff
@@ -29,7 +29,7 @@ defmodule NudgeWeb.Router do
 
   scope "/", NudgeWeb do
     pipe_through [:browser, :with_current_user]
-
+    resources "/site", SiteController
```

Comment:
since some actions are not needed, like delete and update (as of now), ideally, we should mention it here.
you can use https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4-options `only` or `except` here for the same